### DISCUSSION TITLE
Suricata logstash error Exiting: index management requested but the Elasticsearch output is not configured/enabled

### DISCUSSION THREAD

**Discussion by k-alki** [Security-Onion-Solutions/securityonion#11383] [2.4]
### Version

2.4.10

### Installation Method

Network installation on Ubuntu

### Description

configuration

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

4

### RAM

16

### Storage for /

250Go

### Storage for /nsm

50

### Network Traffic Collection

tap

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hello !
I get this error when i want to execute sudo filebeat setup on my client : 
Exiting: index management requested but the Elasticsearch output is not configured/enabled

this is my filebeat.yml : 

filebeat.inputs:
- type: filestream
  id: my-filestream-id
  enabled: false
  paths:
    - /var/log/*.log

filebeat.config.modules:
  path: ${path.config}/modules.d/*.yml
  reload.enabled: false

output.logstash:
  hosts: ["[onionsecurity IP]:5055"]

processors:
  - add_host_metadata:
      when.not.contains.tags: forwarded
  - add_cloud_metadata: ~
  - add_docker_metadata: ~
  - add_kubernetes_metadata: ~

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by dougburks** [comment]
What exactly are you trying to do?

Is this related to your other discussion at https://github.com/Security-Onion-Solutions/securityonion/discussions/11378?
